service.subtitles.titlovi-com-official/service.py

```diff
@@ -0,0 +1,593 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+import re
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+import xbmcvfs
+import requests
+import simplecache
+import time
+import unicodedata
+import io
+import codecs
+from datetime import datetime, timedelta
+from urllib.parse import parse_qs
+from urllib.parse import quote_plus
+from zipfile import ZipFile
+
+addon = xbmcaddon.Addon()
+video_info = xbmc.InfoTagVideo()
+
+author = addon.getAddonInfo('author')
+script_id = addon.getAddonInfo('id')
+script_name = addon.getAddonInfo('name')
+version = addon.getAddonInfo('version')
+get_string = addon.getLocalizedString
+
+script_dir = xbmc.translatePath(addon.getAddonInfo('path'))
+profile = xbmc.translatePath(addon.getAddonInfo('profile'))
+libs_dir = xbmc.translatePath(os.path.join(script_dir, 'resources', 'lib'))
+temp_dir = xbmc.translatePath(os.path.join(profile, 'temp', ''))
+
+player = xbmc.Player()
+
+sys.path.append(libs_dir)
```

Comment:
sorry this is not allowed. Please use relative module imports:

`from x.x.x import y`

Reply:
Fixed

Reply:
It is still here as far as I can see

Reply:
Sorry about that. I have now pushed correct fix.